### PR TITLE
Factors affecting the performance of systems during training in deep learning.

### PR DESCRIPTION
**Pull request by KayTee24**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [X] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [X] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [X] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [X] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [X] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [X] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [X] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [X] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [X] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [X] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [X] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [X] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [X] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [X] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [X] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [X] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by KayTee24** [comment]
This closes #3630

**Comment by duncandegwa** [comment]
Hello @hectorkambow  @ahmadmardeni1 I would like to review this article.

**Comment by ahmadmardeni1** [comment]
Go ahead @duncandegwa

**Comment by lalith1403** [comment]
plagiarism check done ✔️

**Comment by lalith1403** [comment]
plagiarism check done ✔️

The introduction and key takeaways paragraph needs to be re-written. They are completely plagiarized. Apart from that, the entire article is original. Thank you.

**Comment by KayTee24** [comment]
Hello, Good day to you.

Are you making reference to
https://github.com/section-engineering-education/engineering-education/issues/3630#issue-985236042

If your last mail meant I should review my Topic proposal, then I just have
done that.
Thanks & Regards

**Comment by duncandegwa** [comment]
Hello @KayTee24  Add author details I think this is your first article. Right?

**Comment by hectorkambow** [comment]
@duncandegwa Have you checked in with me before doing a review via Slack 🤔  

cc @section-engineering-education/content-moderator-team 
cc @ahmadmardeni1

**Comment by hectorkambow** [comment]
> Hello @hectorkambow @ahmadmardeni1 I would like to review this article.

This needs to be approved via slack 👍 @duncandegwa

**Comment by KayTee24** [comment]
> Hello @KayTee24  Add author details I think this is your first article. Right?

Yeah, it is.

**Comment by hectorkambow** [comment]
@section-engineering-education/peer-reviewers

Is anyone available to review this PR?

Thank you

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+### INTRODUCTION
+Technology is fast growing in the world today. This recency in growth is enhancing the livelihoods of humans and their environment. This includes the building of many self-driven cars and some smart digital systems.
+A few problems in the understanding of its working algorithm have arisen. 
+   
+   These problems have triggered developers to ask questions like: "
+
+* What is the design algorithm enhancing deep learning?
+* Why are recent developed systems able to recognize humans and their environment?
+* How can devices move accurately without human beings controlling them?
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:9]: For bulleted points, always use `-`

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+      An excessive volume of information (data sets) can be trained. This uses learning algorithms in deep learning. Deep learning systems tend to gather functional facts from larger datasets. This experience (learning) is in relation to human ways of getting smarter.
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:11]: This statement is unclear

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+### INTRODUCTION
+Technology is fast growing in the world today. This recency in growth is enhancing the livelihoods of humans and their environment. This includes the building of many self-driven cars and some smart digital systems.
+A few problems in the understanding of its working algorithm have arisen. 
+   
+   These problems have triggered developers to ask questions like: "
+
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:6]: We can have a more briefer intro.

Give an insight of what the reader is going to learn, what is this topic about, why should they learn, see if beginners can understand this article. And, don' try to beat around the bush, directly get to the point.

Have a look at other articles to get an idea of how intro could be

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+* What is the design algorithm enhancing deep learning?
+* Why are recent developed systems able to recognize humans and their environment?
+* How can devices move accurately without human beings controlling them?
+
+      An excessive volume of information (data sets) can be trained. This uses learning algorithms in deep learning. Deep learning systems tend to gather functional facts from larger datasets. This experience (learning) is in relation to human ways of getting smarter. 
+
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:12]: Add `Table of contents` section (Make sure to use markdown hyperlinks) and `Pre-requisities` section.

Have a look at other articles

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+#### Do you think deep learning will make current technologies better?
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:13]: Start with `###` markdown header

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+Deep learning is, without a doubt, at the pinnacle of the most current technological hierarchy. The ability to collect data is crucial for systems (information). The data acquired enables exact forecasting and establishes the DL. As a prediction analysis, comprehending the facial recognition algorithm. This analysis functions by collecting a group of consumer behavior data sets. The intelligence and accuracy rate of deep learning are immense, and they are being checked at all levels. Humans and their environments have been great benefactors of this recent smart technology.
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:15]: There is no flow between lines. It looks very unclear.

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+ The ability to fine-tune the budget and the considered problem are two enhancing variables in Deep Learning. Any optimizer’s accuracy is measured using these factors. 
+
+### Some Determinants That Transform Deep Learning Systems' Performance
+
+#### i. Taking into account the issue (considered problem) that the system will be tasked with resolving;
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:20]:
```suggestion
#### Taking into account the issue (considered problem) that the system will be tasked with resolving; 
```

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+ The ability to fine-tune the budget and the considered problem are two enhancing variables in Deep Learning. Any optimizer’s accuracy is measured using these factors. 
+
+### Some Determinants That Transform Deep Learning Systems' Performance
+
+#### i. Taking into account the issue (considered problem) that the system will be tasked with resolving; 
+          Simply said, this refers to the task for which the system will be created. This problem, which the system must solve, is the most crucial consideration when modeling an algorithm.
+ ######  Basic problems
+* What kind of system should be built? 
+* How will the developed algorithm handle the immediate user’s demands?
+* How easily will re-engineering be made possible?
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:25]: Use `-`

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+#### ii. Evaluation difficulties of default hyperparameters for multiple optimisers.
+          Poor accuracy is obtained here as a result of low awareness from training the system data sets. The Adams optimiser is used in this deep system algorithm, which is adaptable. 
+
+#### iii. Technical expertise in deep learning system network design.
+
+          The demand and need for deep learning experts is highly recommended here. Experience is needed to increase the learning rate and improve the system's performance.
+
+#### iv. Which optimiser suits the deep network best?
+          According to research, the Adam optimiser offers the highest and best performance rate. The ability to outperform other approaches (optimisers) is lacking.
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:35]: Remove these additional tab spaces. And, you can remove these roman numerals as well.

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+### Some Determinants That Transform Deep Learning Systems' Performance
+
+#### i. Taking into account the issue (considered problem) that the system will be tasked with resolving; 
+          Simply said, this refers to the task for which the system will be created. This problem, which the system must solve, is the most crucial consideration when modeling an algorithm.
+ ######  Basic problems
+* What kind of system should be built? 
+* How will the developed algorithm handle the immediate user’s demands?
+* How easily will re-engineering be made possible?
+
+#### ii. Evaluation difficulties of default hyperparameters for multiple optimisers.
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:27]: Beginners might not be aware of these terms. Please make sure to explain them

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+          Poor accuracy is obtained here as a result of low awareness from training the system data sets. The Adams optimiser is used in this deep system algorithm, which is adaptable.
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:28]: Don't think we can state this fact without linking adequate research papers or articles.

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md (hunk 9 above)
Comment (srishilesh) [reply content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:35]: ` According to research`

Please link the paper

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+The network's training methods are the generator and discriminator progression layers. A convolutional neural network (CNN)-based classifier is made up of two layers. These layers, which are referred to as the earlier and later layers,  The final layer is made up of two levels: objects and parts. The system associates with the layers and emerges when it notices this sub-layer. There is a lot of color dictation in the early levels.
+ Research has highlighted the importance of performance rate categorization in CNN neurons. Some deep neural networks are trained to create scenes, while others are trained to recognize them. This helps to improve the data flow in the system. 
+Manipulation of GAN's unit-producing outlets is complicated by artificial pruning. Trained images, like adversarial examples, are developed, categorized, and highlighted on certain neurons. These neurons are treated in some way in order to attain precision.
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:46]: Why are these points more aligned towards CNN or GAN? Shouldn't it be generalized?

Explanation of various terminologies are missing

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+In a deep network, the degree of freedom determines whether or not success (accuracy) is possible. During the training process, the theory has aspects to consider. Gordon's Escape Theorem is generalized by the theory that establishes a powerful theorem. This theorem can be applied to any set that is well-known. In a successful chance, the principal outcome emphasizes the presence of a section transition. It's best to keep dimensionality as low as possible while initializing parameters. Dimensionality influences the sub-level set of network layers.
+The theory (lottery subspaces) seeks to repeat data from prior training experiments. It then establishes the settings for a low-dimensional projection. It compares the parameters to such a trajectory's top-dimensional principal components. The result of this theoretical hypothesis is that both postulates are true. For the same compression, subspace-confined neural networks could even outperform lottery tickets.
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:60]: It looks very technical, and is not beginner friendly.

Please make sure to explain the necessary keywords and maintain a flow between sentences.

File: content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+* Technical know-how
+* Which optimiser suits the learning algorithm best?
+
+There are some steps to aim at in order to achieve a perfect performance rate in a deep learning system. Factors like;
+* The functionality of individual layers in the deep network system.
+* The efficiency of soft convolutional inductive biases in vision transformation improvement.
+* Suitable degrees of freedom to aim at in the deep network while training.
+
+
+### Diagram Demonstrating The Different Scenarios One Can Fall Into When Configuring The Learning Rate.
```
Comment (srishilesh) [inline content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md:78]: Is this subtopic incomplete? Why is this after conclusion? Have you missed out something?

**Review by srishilesh — CHANGES_REQUESTED** [review]
@KayTee24 

Welcome to EngEd program!

I've suggested some major changes to article. Please make sure to address every points. Here are a few important things you have missed out:
- A hero image with dimension 600x400
- Lack of flow and clarity in explaining points
- Not enough research supporting your points. Add links wherever while stating facts or research.
- Markdown doesn't look good.
- Terminologies not explained
- Table of contents, Prerequisities and References sections missing.

Overall, I feel this article needs some major improvements, since it's not very beginner friendly. I would suggest you to take a step back, have a look at other Section articles.

If needed, you can attend [this](https://developers.google.com/tech-writing) course by Google on technical writing.

To allow you to make changes, we will be closing this PR for now. You can work on these changes in your forked repository. Once you're done, create a new PR and ping me.

Thanks.

**Pull request closed without merging**